lib/commands/update-kendoui.ts

```diff
@@ -27,31 +46,30 @@ class UpdateKendoUICommand implements ICommand {
 			}
 
 			var packages: Server.IKendoDownloadablePackageData[] = _.filter(<Server.IKendoDownloadablePackageData[]>this.$server.kendo.getPackages().wait(), p => !p.NeedPurchase);
+			if(options.verified) {
+				packages = _.filter(packages, pack => _.any(pack.VersionTags, tag => tag.toLowerCase() === UpdateKendoUICommand.VERIFIED_TAG));
+			}
 
-			this.$logger.out("You can download and install the following Kendo UI Core or Kendo UI Professional packages.");
-			var table = new Table({
-				head: ["#", "KendoUI", "Version", "Tags"],
-				chars: {'mid': '', 'left-mid': '', 'mid-mid': '', 'right-mid': ''}
-			});
-			_.each(packages, (update: Server.IKendoDownloadablePackageData, idx: number) => table.push([
-					(idx + 1).toString().cyan.toString(),
-					update.Name,
-					update.Version,
-					(update.VersionTags || []).join(", ").cyan.toString()
-				]));
-			this.$logger.out(table.toString());
-
-			var schema : IPromptSchema = {
-				type: "input",
-				name: "packageIdx",
-				message: "Enter the index of the package that you want to install",
-				validate: (value: string) => {
-					var num = parseInt(value, 10);
-					return !isNaN(num) && num >= 1 && num <= packages.length ? true : "Valid values are between 1 and " + packages.length;
-				}
-			};
+			if(options.core) {
+				packages = _.filter(packages, pack => pack.Name === UpdateKendoUICommand.KENDO_CORE);
+			}
 
-			var choice = this.$prompter.get([schema]).wait();
+			if(options.professional) {
+				packages = _.filter(packages, pack => pack.Name === UpdateKendoUICommand.KENDO_PROFESSIONAL);
+			}
+
+			var downloadUri: string;
+			if(options.latest) {
+				var latestPackage = _.first(packages);
```

Comment:
It's a little bit strange to me - `var latestPackage = _.first(packages)` :smile: Is it possible packages to be empty []? If yes, `latestPackage.DownloadUrl` will throw an error.